TODO.md

```diff
@@ -15,10 +6,36 @@
 6. [x] click on download button, get the app in the chat, the download button changes to "downloading.." and
```

Comment:
that's not the current state but i am also wondering if it maeks sense to keep all this in the TODO when we also have issues.

Reply:
It is true that with the gh-project and repo-issues, we could remove this file, but for me, it's a more usable overview that I like to use personally.  We should keep it in sync with the current work though, that's right...